doc/source/serve/advanced-guides/performance.md

```diff
@@ -103,6 +103,7 @@ There are handful of ways to address these issues:
 2. Try batching your requests. See [Dynamic Request Batching](serve-performance-batching-requests).
 3. Consider using `async` methods in your callable. See [the section below](serve-performance-async-methods).
 4. Set an end-to-end timeout for your HTTP requests. See [the section below](serve-performance-e2e-timeout).
+5. Set keep alive timeout for uvicorn server on Serve's HTTP proxy. See [the section below](serve-performance-keep-alive-timeout).
```

Comment:
Is there a better place to link this? I don't think the keep alive timeout is related to performance issues. Maybe in [the HTTP guide](https://docs.ray.io/en/latest/serve/http-guide.html)?

Reply:
Yea, I feel it's related to the request timeout so put it here. Will move to http guide